screen/RoomMaker/components/GameLevel.js

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect, useState } from 'react';
+import { View } from 'react-native';
+import PropTypes from 'prop-types';
+
+import Input from '../../../common/components/Input';
+
+const GameLevel = ({ onInputChange, mode }) => {
+  const [isSwitched, setIsSwitched] = useState(false);
+
+  useEffect(() => {
+    if (mode !== 'servival') {
+      setIsSwitched(!isSwitched);
```

Comment:
```suggestion
      setIsSwitched((prevState) => !prevState);
```
직접적으로 state 훅에 할당된 state에 논리연산자를 할당 할 시에는 버그를 유발 할 수 있을거 같습니다.